src/components/AwardItem/AwardItem.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '../../index.css';
+import styles from './AwardItem.module.css';
+
+/**
+* Component to add title and horizontal line
+* @param {Int} Year Number of year (Int)
+*/
+const AwardYear = ({ Year }) => {
+  return (
+    <div>
+      <h3 className={`${styles.h3__year}`}>{Year}</h3>
+      <hr width="95%" />
+    </div>
+  );
+};
+AwardYear.defaultProps = {
+  Year: 9999,
+};
+
+/**
+* To add a singular award
+* @param  {string} RocketName Name of Rocket (Str)
```

Comment:
```suggestion
* @param  {string} RocketName Name of Rocket
```
Can probably remove the old way you specified the type considering that the JSDoc syntax does it itself.